src/openwrt/view/admin_thread/thread_scan.htm

```diff
@@ -74,12 +74,9 @@ <h2><%:Join Network: Thread Scan%></h2>
 		<div class="table">
 			<div class="tr table-titles">
 				<div class="th col-1 center"><%:RSSI%></div>
-				<div class="th col-3 center"><%:Network Name%></div>
```

Comment:
We probably want to use `Discover`, the commissioning process requires `network name` and `extended panid` for computing `pskc`.

Reply:
Any insight about which places in the code to modify? I'm not very familiar with this `openwrt` functionality.

Reply:
We can address this in a separate PR.